### DISCUSSION TITLE
Frigate Developers Needed

### DISCUSSION THREAD

**Discussion by ImAsht** [blakeblackshear/frigate#14488] [General]
Hi, I'm exploring the feasibility of using a modified version of Frigate customized to work with a suite of product lines we are putting together for the security camera industry.

The project has three objectives: 1. Customize an out-of-the-box solution to work with our inventory of Reolink and higher-end Panasonic cameras. 2. Enhance Frigate's UI with additional features. 3. Contribute to and improve the Frigate community with code contributions.

We are looking for developers with whom we can exchange ideas. If anyone is interested or knows of anyone who may be interested, please reach out to me.

**Comment by Sceptersax** [comment]
@ImAsht, If you need an edge device to host your platform, let me know. What i have in mind is a small device that has an m.2 form factor accelerator. It has it's own OS and docker platform so you can easily update, upgrade and add additional containers with ease. The edge device is made by a company called Latticework and they'd be happy to help you customize the software too if you'd like. They also offer cloud storage if you need it:

support@latticeworkinc.com

**Reply by ImAsht** [reply]
Thank you. I checked out their site. I will contact them next week to see if they meet any of our needs.

**Comment by moandcompany** [comment]
I think it would be worth clarifying if your intent is to distribute Frigate as a commercial solution or part of a commercial solution, as well as whether or not your interest in finding developers is for-hire, sponsoring their contributions to the open-source project, or if you are simply looking to discuss potential features or use cases for your proposed modified version of Frigate.

**Reply by ImAsht** [reply]
Hi, and great question, as it is the central theme of the initial objective. Ideally, we see:

1. We need a pre-packaged solution tailored to our line of cameras to facilitate and simplify the on-site installation process.
2. Enhance Frigate's UI and functionality in a number of ways. We have general goals here and will formalize a development RoadMap once we have a team and reach a consensus on tasks and priorities.
3. We are in need of developers, particularly a primary developer who is well-versed in all aspects of Frigate. Contract work for various aspects, including UI, is also on the horizon as we progress through the RoadMap. 
4. Our dev work is for hire. Our business model is likely similar to open-source SaaS solutions. Software (Frigate) is included in our work free of charge. Revenue is generated from installations and post-sale services. Some aspects are still being explored, and any feedback on an ideal business model that is equitable to all is welcome.
5. We'd like to work out a way to white-label Frigate, and I see no reason enhancements we finance can't benefit the community overall with code contributions. This, too, needs to be formalized once a team is assembled. 

I believe that covers everything! Interested devs please reach out to me asap. Best, C.

**Reply by ImAsht** [reply]
Hi everyone! - Is anyone interested in this freelance work?

**Reply by Sceptersax** [reply]
Contact Latticework. It turns out the platform already has Frigate integrated. And they can work with you.

**Reply by ImAsht** [reply]
Latticework's VAI rep said they have an in-house custom solution and no association with Frigate. I asked about customizing a white-labeled solution, but he didn't seem interested in pursuing the topic further. I thank you for the lead, though. If anything else comes to mind, please send it over. Thank you; I appreciate your time!

**Reply by Sceptersax** [reply]
That's crazy. The gate keepers don't have a clue... this guy knows what's up:

Mac.sewell@cnt.latticeworkinc.com
